docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module

```diff
@@ -256,6 +256,34 @@ function cgov_core_form_alter(&$form, FormStateInterface $form_state, $form_id)
         ($user->hasPermission('administer permissions') || $user->hasPermission('edit user roles'));
     }
   }
+
+  // Set up a custom submitter for media edit forms so we can force them to the list of
+  // items rather than the latest revision form.
+  if ($form_id === 'media_cgov_file_edit_form' || $form_id === 'media_cgov_image_edit_form' || $form_id === 'media_cgov_contextual_image_edit_form'
+    || $form_id === 'media_cgov_infographic_edit_form' || $form_id === 'media_cgov_video_edit_form') {
```

Comment:
Infographic and video act as content pages so they should act like any other piece of content. Was there a reason that you added these?